pilot/pkg/model/authentication.go

```diff
@@ -18,10 +18,23 @@ import (
 	"fmt"
 	"net/url"
 	"strconv"
+	"time"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/auth"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	"github.com/gogo/protobuf/types"
 
 	authn "istio.io/api/authentication/v1alpha1"
 )
 
+const (
+	// CARootCertPath is the path of ca root cert that envoy uses to validate cert got from SDS service.
+	CARootCertPath = "/etc/istio/certs/ca-root-cert.pem"
```

Comment:
SDS should populate the ca-root-cert.pem as well - there is no point in using both Secrets and sds.

Reply:
once envoy's change to fetch root cert through sds is done, this could be removed